docs/benchmark.md

```diff
@@ -12,86 +12,202 @@ It's important to look at the units than at the percentages. In general, RxDogTa
+| RxDogTagAndroidPerf.flowable_complex[enabled=true,times=1,guardedDelegateEnabled=true]             |     131,163ns |        0.131ms | 23922.53% |
+| RxDogTagAndroidPerf.flowable_simple[enabled=true,times=1,000,guardedDelegateEnabled=true]          |     152,370ns |        0.152ms | 27806.59% |
+| RxDogTagAndroidPerf.flowable_e2e[enabled=true,times=1,000,guardedDelegateEnabled=true]             |     383,229ns |        0.383ms | 70088.46% |
+| RxDogTagAndroidPerf.flowable_e2e[enabled=true,times=1,000,guardedDelegateEnabled=false]            |     390,417ns |        0.390ms | 71404.95% |
+| RxDogTagAndroidPerf.flowable_complex[enabled=false,times=1,000,guardedDelegateEnabled=false]       |     470,885ns |        0.471ms | 86142.67% |
+| RxDogTagAndroidPerf.flowable_complex[enabled=true,times=1,000,guardedDelegateEnabled=true]         |     592,499ns |        0.592ms | 108416.30% |
+| RxDogTagAndroidPerf.flowable_complex[enabled=true,times=1,000,guardedDelegateEnabled=false]        |     718,021ns |        0.718ms | 131405.68% |
+| RxDogTagAndroidPerf.flowable_e2e[enabled=false,times=1,guardedDelegateEnabled=false]               |     870,794ns |        0.871ms | 159386.08% |
+| RxDogTagAndroidPerf.flowable_e2e[enabled=false,times=1,000,guardedDelegateEnabled=false]           |   1,060,676ns |        1.061ms | 194163.00% |
+| RxDogTagAndroidPerf.flowable_complex[enabled=false,times=1,guardedDelegateEnabled=false]           |   1,456,641ns |        1.457ms | 266684.07% |
+| RxDogTagAndroidPerf.flowable_simple[enabled=false,times=1,000,000,guardedDelegateEnabled=false]    |  23,566,147ns |       23.566ms | 4316044.14% |
+| RxDogTagAndroidPerf.flowable_simple[enabled=true,times=1,000,000,guardedDelegateEnabled=false]     |  26,045,210ns |       26.045ms | 4770084.98% |
+| RxDogTagAndroidPerf.flowable_simple[enabled=true,times=1,000,000,guardedDelegateEnabled=true]      | 158,885,276ns |      158.885ms | 29099767.40% |
+| RxDogTagAndroidPerf.flowable_e2e[enabled=true,times=1,000,000,guardedDelegateEnabled=true]         | 560,666,826ns |      560.667ms | 102686131.87% |
+| RxDogTagAndroidPerf.flowable_complex[enabled=true,times=1,000,000,guardedDelegateEnabled=false]    | 593,260,735ns |      593.261ms | 108655712.27% |
+| RxDogTagAndroidPerf.flowable_complex[enabled=false,times=1,000,000,guardedDelegateEnabled=false]   | 629,395,375ns |      629.395ms | 115273778.21% |
+| RxDogTagAndroidPerf.flowable_e2e[enabled=false,times=1,000,000,guardedDelegateEnabled=false]       | 630,688,866ns |      630.689ms | 115510681.32% |
+| RxDogTagAndroidPerf.flowable_e2e[enabled=true,times=1,000,000,guardedDelegateEnabled=false]        | 654,682,512ns |      654.682ms | 119905121.98% |
+| RxDogTagAndroidPerf.flowable_complex[enabled=true,times=1,000,000,guardedDelegateEnabled=true]     | 755,996,898ns |      755.997ms | 138460870.33% |
+
+#### 1_000 items (Observable)
```

Comment:
Same here as we're showing results for 1_000_000 items (which has a separate section anyway)